_static/files/audit-event-examples/R4_read.json

```diff
@@ -0,0 +1,122 @@
+{
+    "resourceType": "AuditEvent",
+    "type": {
+        "system": "http://hl7.org/fhir/audit-event-type",
+        "code": "rest",
+        "display": "RESTful Operation"
+    },
+    "subtype": [
+        {
+            "system": "http://hl7.org/fhir/http-verb",
```

Comment:
We could add a display value to both codings

Reply:
Do I just re-use the values from the `code` element?